scripts/run-examples-from-readme.sh

```diff
@@ -59,23 +57,49 @@ do
     set -e
 
     # Stop at first failure
-    if [ ${exit_code} -ne 0 ]
-    then
+    if [ ${exit_code} -ne 0 ]; then
         echo ""
-        echo -e "\e[31mCommand failed:\e[0m ${command}"
+        echo -e "\e[31mCLI command failed:\e[0m ${command}"
         echo ""
         break
     fi
 
 done < <(grep -E "^\`cargo r --bin iggy -- " README.md)
```

Comment:
Do you think it would be good to have all examples consistent and use always `cargo run` instead of `cargo r`?